bootstrap.py

```diff
@@ -383,13 +414,31 @@ def install_puppet_agent():
     call_yum("install", "puppet")
     enable_service("puppet")
 
-    puppet_conf = open('/etc/puppet/puppet.conf', 'wb')
-    puppet_conf.write("""
+    puppet_major_version = get_puppet_version()
+    if puppet_major_version == 3:
+        puppet_conf_file = '/etc/puppet/puppet.conf'
+        main_section = """
```

Comment:
cosmetic: this makes `main_section` start with a newline, and thus the generated `puppet.conf` starts with two newlines. can we change this to `main_section = """[main]…`?